src/main/java/nl/hsac/fitnesse/fixture/util/selenium/SeleniumHelper.java

```diff
@@ -37,17 +40,10 @@
                     "  rect.right <= (window.innerWidth || document.documentElement.clientWidth));\n" +
             "} else { return null; }";
 
-    private static final String TOP_ELEMENT_AT =
-            "if (arguments[0].getBoundingClientRect) {\n" +
-                    "  var rect = arguments[0].getBoundingClientRect();\n" +
-                    "  var x = (rect.left + rect.right)/2;\n" +
-                    "  var y = (rect.top + rect.bottom)/2;\n" +
-                    "  return document.elementFromPoint(x,y);\n" +
-                    "} else { return null; }";
-
     private final List<WebElement> currentIFramePath = new ArrayList<WebElement>(4);
     private DriverFactory factory;
     private WebDriver webDriver;
+    private AndroidDriver<MobileElement> androidDriver;
```

Comment:
Would it be at all possible to store the androidDriver in the webDriver field? I believe it implements the WebDriver interface. 

It looks like this class could remain pretty much the same as it was then. It would then be the caller's responsibility to cast the driver() to AndroidDriver. But your code requires the caller to call different methods so that seems workable...